### PR TITLE
Redis 6 in builds

### PR DESCRIPTION
**Pull request by DvirDukhan**

### CODE REVIEW

**Comment by swilly22** [comment]
@DvirDukhan circle seems to fail, can you please take a look?

File: Dockerfile [hunk 1]
```diff
@@ -1,34 +1,71 @@
...
+ARG OSNICK=buster
+
+# OS=debian:buster-slim|debian:stretch-slim|ubuntu:bionic
+ARG OS=debian:buster-slim
+
+# ARCH=x64|arm64v8|arm32v7
+ARG ARCH=x64
+
+#----------------------------------------------------------------------------------------------
+FROM redisfab/redis:${REDIS_VER}-${ARCH}-${OSNICK} AS redis
```
Comment (swilly22) [inline Dockerfile:15]: Please try switching to https://hub.docker.com/_/redis

File: Dockerfile [hunk 2]
```diff
@@ -1,34 +1,71 @@
...
+FROM ${OS} AS builder
+
```
Comment (swilly22) [inline Dockerfile:17]: Please add comment, saying something like, running on builder.

File: Dockerfile [hunk 3]
```diff
@@ -1,34 +1,71 @@
...
+ARG OSNICK
+ARG OS
+ARG ARCH
+ARG REDIS_VER
```
Comment (swilly22) [inline Dockerfile:21]: Please add a comment explaining these lines

File: Dockerfile [hunk 4]
```diff
@@ -1,34 +1,71 @@
...
+
+ARG PACK=0
+ARG TEST=0
+
+RUN set -ex ;\
+	if [ "$PACK" = "1" ]; then \
+		python -m RAMP.ramp pack -m ramp.yml -o "build/redisgraph.{os}-{architecture}.{semantic_version}.zip" src/redisgraph.so ;\
+	fi
+RUN set -ex ;\
+	if [ "$TEST" = "1" ]; then TEST= make test; fi
```
Comment (swilly22) [inline Dockerfile:54]: Swap TEST with PACK

File: Dockerfile (hunk 1 above)
Comment (rafie) [reply Dockerfile:15]: OK, just recalled the story behind Redis docker:
The official repo uses https://github.com/docker-library/redis to create their Redis image. We are based on the same repo, with our own tweaks. For instance, we have a scheme that supports other platforms than debian:buster and alpine.
So, in the bottom line, I think we better stay with our version of Redis docker.

**Pull request merged**